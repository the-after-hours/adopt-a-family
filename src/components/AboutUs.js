// Page to create/About Us

import React from 'react';
import links from '../constants/links';

const AboutUs = (props) => {
  return (
    <div>
      <h1>AboutUs page</h1>
      Check us out on <a href={links.github}>Github</a>!
    </div>
  );
};

export default AboutUs;