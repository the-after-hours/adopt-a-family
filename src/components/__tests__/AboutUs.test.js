import React from 'react';
import AboutUs from '../AboutUs';
import renderer from 'react-test-renderer';

describe('AboutUs', () => {
  test('AboutUs snapshot', () => {
    const component = renderer.create(<AboutUs />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
