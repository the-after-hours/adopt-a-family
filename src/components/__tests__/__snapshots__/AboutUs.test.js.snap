// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs AboutUs snapshot 1`] = `
<div>
  <h1>
    AboutUs page
  </h1>
  Check us out on 
  <a
    href="https://github.com/5-gwoap"
  >
    Github
  </a>
  !
</div>
`;
